_posts/2019-07-25-wasm-cpu-profiles.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: post
+title: Generating WebAssembly CPU Profiles in Go
+categories: [wasm]
+tags: [profile, wasm, golang, go]
+fullview: true
+comments: true
+---
+
+Go has had WebAssembly (wasm) support for a while now. But the tooling is still in it's nascent stages. It is straightforward to build a wasm module from Go code. But running tests in a browser is still cumbersome, as it requires some HTML and JS glue to work. And generating a CPU profile isn't even possible since wasm does not have thread support (yet).
```

Comment:
I think the Oxford comma would make this read nicer than all these periods.

Reply:
Done